main.js

```diff
@@ -16,11 +16,11 @@ const {
 
 const packageJson = require('./package.json');
 
+const autoUpdate = require('./app/auto_update');
 const createTrayIcon = require('./app/tray_icon');
-const createTemplate = require('./app/menu.js');
 const logging = require('./app/logging');
-const autoUpdate = require('./app/auto_update');
 const windowState = require('./app/window_state');
+const { createTemplate } = require('./app/menu');
```

Comment:
What's the reasoning behind these order changes? It wasn't alphabetized before, but doesn't get us any closer.

Reply:
Apologies, old habit of mine. I use Sublime Text’s **Sort Lines** feature that does this automatically. For destructured imports this means they’ll show up last because of `{`. Let me know if you want me to revert 👀

Reply:
Since we're making changes, let's sort the right side; the modules we're pulling in.

Reply:
I don’t think that’s a good idea. Preferably one wants to scan the left-hand side using binary search (humans are good at that 😉) for some identifier. In addition, ESLint proposes the same and we can use it once we have `import` support, see: https://eslint.org/docs/rules/sort-imports

There is also https://github.com/kentor/eslint-plugin-sort-requires for `require` which seems to sort left-hand side as well. In practice, it won’t be a big change as we should preferably call our left-hand side the same as the module they come from, hence the preference for named imports.

Reply:
Popping up a level, I like the high-level way the [`order`](https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/order.md) rule from `eslint-plugin-import` works. Built-ins first, then third-party node modules, then your stuff, etc.

It's disheartening that destructuring would mess with our small-scale ordering. But I guess I understand what it's doing now.

Reply:
Thanks for doing more research. Will merge for now, but I am open to automating this down the line.